Source/Scene/Cesium3DTileset.js

```diff
@@ -1276,6 +1282,26 @@ define([
 
     ///////////////////////////////////////////////////////////////////////////
 
+    function applyDebugSettings(tileset, frameState) {
+        // Draw a debug camera in freeze frame mode
+        var enterFreezeFrame = tileset.debugFreezeFrame && !tileset._debugFreezeFrame;
```

Comment:
I kinda want to destroy `_debugCameraFrustum` when we leave freeze frame, but I don't know if the verbosity is worth it; if it wasn't debug code, it would be.